src/types/video.ts

```diff
@@ -217,9 +196,9 @@ export interface ReactVideoProps extends ReactVideoEvents, ViewProps {
   repeat?: boolean;
   reportBandwidth?: boolean; //Android
   resizeMode?: EnumValues<VideoResizeMode>;
-  selectedAudioTrack?: SelectedTrack;
-  selectedTextTrack?: SelectedTrack;
-  selectedVideoTrack?: SelectedVideoTrack; // android
+  selectedAudioTrack?: SelectedTrackType;
```

Comment:
I don't undestand why you did this change ... 
SelectedTrack is an object representing all possible values to select a track where  SelectedTrackType is an enum representing which kind of track selection can be used

Reply:
Hmm I couldn't fully understand your point.
So you want to use typescript's enum value as selectedAudioTrack's type?

Reply:
Ah, mate.
I think I just deleted typescript enum types and replaced them from codegen types to recycle codegen types.
Just naming and few types change for codegen types.
I think there would have no problem?

Let me know your opinion.
@freeboub

Reply:
Here is doc:
selectedAudioTrack
Configure which audio track, if any, is played.

selectedAudioTrack={{
  type: Type,
  value: Value
}}

SelectedAudioTrack is an object
SelectedAudioTrack.type is the enum you use.

Is that clearer ?

Reply:
We use https://github.com/react-native-video/react-native-video/pull/3487/files#diff-70773b05560f0897b24b4baa0ed114158fd50e075c0c76de43cc30b90043eb80R75 this type from 'specs/VideoNativeComponent.tsx'

![스크린샷 2024-01-30 오전 10 38 17](https://github.com/react-native-video/react-native-video/assets/36250002/92bbe3af-ec82-4a91-abbe-2286039e802f)

We cannot use typescript's enum type with codegen. It can't be possible.
So I recycled above codegen type for Selected**Track prop types.

I think you want to consider DX for developers by handling `type` as enum.
I think it can be possible, but I'm not sure. I will have a try.

Reply:
@freeboub Can you check this commit?
I've handled some issue about this comments.
https://github.com/react-native-video/react-native-video/pull/3487/commits/d1a858a590339b48d5430fbe8b8769f8f1921bd6

I think we have to choose between
* remain docs and modify some types like above commit
* change docs and give up enum types

Let me know your opinion. that commit can be reverted.